test/e2e/mock-e2e.js

```diff
@@ -161,7 +161,9 @@ async function setupMocking(
   });
 
   await server
-    .forPost(`${SECURITY_ALERTS_PROD_API_BASE_URL}/validate/${chainId}`)
+    .forPost(
+      `${SECURITY_ALERTS_PROD_API_BASE_URL}/validate/0x${chainId.toString(16)}`,
+    )
```

Comment:
this fixes the mock, as now we use the hex value of the chainId, which is what the API call uses